server-x/README.md

```diff
@@ -6,7 +6,7 @@ See the `Build` section for more details on how to build the image.
 
 Once built, you can run the server in the same manner as when using the Keycloak.X distribution by passing any command-line argument:
 
-    docker run --name keycloak -p 8080:8080 \
+    docker run --rm --name keycloak -p 8080:8080 \
```

Comment:
We shouldn't add the "--rm" option here, as someone less observant could copy/paste this and by mistake it would destroy their currently running instance

Reply:
I'm wondering how can this happen?
Named containers needs to be removed before another one can start.

We thought this is an improvement since "someone less observant" will not leave a stopped container dangling.

But not fussy on this detail, following the suggestion to speed up the process 👍